bundles/af-core/src/test/resources/form/panel/exporter-rules-panel.json

```diff
@@ -6,6 +6,7 @@
   "name":"abc",
   "visible":true,
   "enabled":true,
+  "readOnly":false,
```

Comment:
readOnly, when not defined should not come, hence:
```
 @JsonInclude(JsonInclude.Include.NON_NULL)
    @Nullable
```
  Defaults should be defined in core.

Reply:
+1. Can you fix this ? Default values would be define by core, else author would never understand what was defined and what was not. Also, this bloats the json model